book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
```

Comment:
- [ ] please kindly add a method for adding rentals. here, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```